config/crd/apirules.gateway.crd.yaml

```diff
@@ -69,12 +69,12 @@ spec:
                             properties:
                               jwks_urls:
                                 items:
-                                  pattern: ^(https://|file://).*$
+                                  pattern: ^(\w+:)?(\/\/)?[^\s]+$
```

Comment:
Is there a reason why we are not using `^(http://|https://|file://).*$`, but open up the validation?

Reply:
yes, internal kyma issue: 4240

Reply:
since we decided we do not go with non-scheme URLs for not and clarify with customer, I will revert the change to support just deterministic schemes: http://, https:// or file://